src/db/oracledb/jobs-dao.js

```diff
@@ -0,0 +1,40 @@
+import _ from 'async-dash';
+
+import { parseQuery } from 'utils/parse-query';
+import { getConnection } from './connection';
+import { contrib } from './contrib/contrib';
+
+/**
+ * Queries data source for raw person data and passes it to the serializer
+ *
+ * @param {string} internalId internal ID of person to select
+ * @param {object} query query parameters passed in with the request
+ * @returns {Promise<object>} Raw job data from data source
+ */
+const getJobs = async (internalId, query) => {
```

Comment:
Some fields like `fullTimeEquivalency`, `appointmentPercent`, etc are parsed as strings now but they should be numbers. I'm not sure did I catch this error at other endpoints before, can you also make sure these number fields are parsed corretly?

Reply:
I did the conversion manually since I want to get persons done. I'll work on the util we discussed later.